sktime/tests/_config.py

```diff
@@ -275,8 +276,25 @@
     Imputer: {"method": "mean"},
     HampelFilter: {"window_length": 3},
     OptionalPassthrough: {"transformer": BoxCoxTransformer(), "passthrough": True},
+    NetworkPipelineForecaster: {
+        "steps": [
+            (
+                "imputer",
+                Imputer(method="drift"),
+                {"fit": {"Z": "original_y"}, "predict": None, "update": None},
+            ),
+            (
+                "forecaster",
+                NaiveForecaster(strategy="mean"),
+                {
+                    "fit": {"y": "imputer", "X": "original_X", "fh": "original_fh"},
+                    "predict": {"fh": "original_fh"},
+                    "update": {"y": "original_y"},
```

Comment:
Why would we want to update the estimator without prior transformation when we used transformations in fitting?

Reply:
this is an oversight, I'll fix it.